modules/komoonaBidAdapter.js

```diff
@@ -1,117 +1,123 @@
-import Adapter from 'src/adapter';
-import bidfactory from 'src/bidfactory';
-import bidmanager from 'src/bidmanager';
 import * as utils from 'src/utils';
-import { ajax } from 'src/ajax';
-import { STATUS } from 'src/constants';
-import adaptermanager from 'src/adaptermanager';
+import { registerBidder } from 'src/adapters/bidderFactory';
 
+const BIDDER_CODE = 'komoona';
 const ENDPOINT = '//bidder.komoona.com/v1/GetSBids';
+const USYNCURL = '//s.komoona.com/sync/usync.html';
+
+export const spec = {
+  code: BIDDER_CODE,
+
+  aliases: ['komoona'],
```

Comment:
This is just for adapters which have multiple names... since this is the same as your `BIDDER_CODE`, you can just delete it.

Reply:
Removed from adapter file, thanks from the comment